cypress/integration/common/i-can-enter-use-information/aviation.spec.ts

```diff
@@ -84,7 +77,7 @@ export const iCanGoBackAndEditMyAviationUse = (purpose: Purpose): void => {
   whenIClickBack();
   iCanEditMyPersonalDetails();
   whenIClickBack();
-  iCanEditMyAdditionalUsesChoice(AdditionalUses.NO);
+  iCanEditMyNUses(1);
```

Comment:
Because the UX is going from a "additional uses, y/n" pattern to presenting the user with the existing uses on the stack.